src/Microsoft.Diagnostics.EventFlow.Inputs.MicrosoftLogging/LoggerInput.cs

```diff
@@ -72,16 +73,12 @@ public void Dispose()
             this.subject.Dispose();
         }
 
-        private void InvokeAndReport(Func<DictionaryExtenstions.AddResult> action)
+        private void AddPayloadProperty(EventData eventData, string key, object value)
```

Comment:
same here: you probably can remove this method, since it just proxies PayloadDictionaryUtilities.AddPayloadProperty